llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -614,11 +614,6 @@ bool CompileTimePropertiesPass::transformSYCLPropertiesAnnotation(
   std::string NewAnnotString = "";
   auto Properties = parseSYCLPropertiesString(M, IntrInst);
   for (auto &Property : Properties) {
-    // sycl-alignment is converted to align on
```

Comment:
Can you please elaborate what changed here and why we are not dropping this here anymore?

Thanks

Reply:
removed. Will do this in a seperate PR